### DISCUSSION TITLE
No information about useConfig

### DISCUSSION THREAD

**Discussion by mirecatercero** [shuding/nextra#1087] [General]
Hello,
 
I am trying to configure dynamic SEO tags and there is no information on how to use the useConfig hook (404 error btw). I would like to have a meta description and title for each page.

Also, I would like to generate my MDX pages dynamically. I have read the "Remote Content" section but the MDX are defined and what I want to do is generate everything from my server. Is that possible?

Thank you very much!

**Comment by SincerelyFaust** [comment]
Regarding the SEO tags and frontmatter (meta description and title for each page), here are some examples on how you can accomplish what you want:

For docs theme:
> - SEO
> https://github.com/dahliaOS/documentation/blob/main/theme.config.tsx#L100 (Line 100 and everything below)
> - Frontmatter
> https://raw.githubusercontent.com/dahliaOS/documentation/main/pages/build/buildroot.mdx (Peep the text inbetween --- tags at the top of the file, you need to do this for each page)

For blog theme:
> - SEO
> https://github.com/dahliaOS/blog/blob/main/theme.config.tsx#L44 (Line 44 to 144)
> - Frontmatter
> https://raw.githubusercontent.com/dahliaOS/blog/main/pages/posts/first-post.mdx (Peep the text inbetween --- tags at the top of the file, you need to do this for each page)

**Comment by mirecatercero** [comment]
Thank you very much!!
